engine/src/main/java/org/destinationsol/game/screens/CameraKbControl.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.destinationsol.game.screens;
+
+import org.destinationsol.GameOptions;
+import org.destinationsol.ui.SolUiControl;
+
+import java.util.List;
+
+class CameraKbControl {
```

Comment:
Would it be worth writing out this class name fully so `CameraKeyboardControl` ? No real harm in spelling it out

Reply:
Functionality and consistence with already existing code in this PR, renaming in the next PR

Reply:
That may be the case, but since you have created this class yourself in this PR I think it better that the name be fixed before the PR is merged.

Reply:
> consistence with already existing code

Reply:
Here is the thing: If you make these small changes as separate commits in PRs that already touch a part of the codebase, they will easily get merged. However, if you keep putting them off and then send them out in a mega PR, they will inevitable take way more time.

So, I will *insist* you make the change here, even more so because you introduced this class in the first palce :)